common/keyboard.go

```diff
@@ -253,6 +251,36 @@ func (k *Keyboard) modifierBitFromKeyName(key string) int64 {
 	return 0
 }
 
+func (k *Keyboard) comboPress(keys string, opts *KeyboardOptions) error {
+	if opts.Delay != 0 {
+		t := time.NewTimer(time.Duration(opts.Delay) * time.Millisecond)
+		select {
+		case <-k.ctx.Done():
+			t.Stop()
+		case <-t.C:
+		}
+	}
+
+	kk := []string{"+"}
+	if keys != "+" {
+		kk = strings.Split(keys, "+")
+	}
+	for _, key := range kk {
+		if err := k.down(key); err != nil {
+			return fmt.Errorf("cannot do key down: %w", err)
+		}
+	}
+
+	for i := len(kk) - 1; i >= 0; i-- {
```

Comment:
Is there a reason why this is not a `range` as the above `down` loop?

Reply:
It's going backwards:

a->b->c
c->b->a

Reply:
Yeah sure, nice little trick 🙂

Reply:
That trick is not intuitive at all IMO. The previous C-style `for` was clear and straightforward.

Please let's not point out style issues in reviews, unless the reviewer can point out an objective benefit of the change.